README.md

```diff
@@ -36,8 +46,6 @@ import runVrt from '@magiclab/vrt-runner';
 runVrt({
     cwd,
     output,
```

Comment:
please return teamcity and options to the readme as well.

When you edit files - please refer to diff change in github diff online - it help you to see produced diff and remove unnecessary changes.